src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java

```diff
@@ -110,6 +111,18 @@ public EditPersonDescriptorBuilder withPostalCode(String postalCode) {
     }
 
     /**
+     * Sets the {@code PostalCode} of the {@code EditPersonDescriptor} that we are building.
+     */
+    public EditPersonDescriptorBuilder withDeadLine(String deadLine) {
+        try {
+            ParserUtil.parseDeadLine(Optional.of(deadLine)).ifPresent(descriptor::setDeadLine);
+        } catch (IllegalValueException ive) {
+            throw new IllegalArgumentException("postal code is expected to be unique.");
```

Comment:
should reflect deadline, not postal code

Reply:
fixed